README.md

```diff
@@ -65,19 +71,19 @@ from mosec.mixin import MsgpackMixin
 logger = get_logger()
 ```
 
-Then, we **build an API** to generate the images for a given prompt. To achieve that, we simply inherit the `MsgpackMixin` and `Worker` class, then override the `forward` method. Note that the input `data` is by default a JSON-decoded object, but `MsgpackMixin` will override it to use [msgpack](https://msgpack.org/index.html) for the request and response data, e.g., wishfully it receives data like `[b'a cut cat playing with a red ball']`. Noted that the returned objects will also be encoded by the `MsgpackMixin`.
+Then, we **build an API** for clients to query a text prompt and obtain an image based on the [stable-diffusion-v1-5 model](https://huggingface.co/runwayml/stable-diffusion-v1-5). To achieve that, we simply inherit the `Worker` and `MsgpackMixin` class, then initialize the model in the `__init__` method and override the `forward` method. By default, the input `data` of the `forward` method should be a JSON-decoded object. Using `MsgpackMixin` will override it to use the faster and smaller [msgpack](https://msgpack.org/index.html) for both request and response data. In this example, the `forward` method will wishfully receive a _list_ of string, e.g., `['a cute cat playing with a red ball', 'a man sitting in front of a computer', ...]`, aggregated from different clients for _batch inference_, improving the system throughput. Also note that the returned objects will be encoded by the msgpack protocol as well, due to the use of `MsgpackMixin`.
```

Comment:
This paragraph is a little bit long. We can split it into two.

Also, we use msgpack because we need to return an image, which is bytes. But JSON doesn't support bytes unless you encode it with base64 (which makes it longer).

Reply:
Maybe we can use the GitHub Flavor README note block like:
```
> **Note**
>
> Long Descriptions here.
```

> **Note**
>
> Long Descriptions here.

Reply:
I feel this "Note" sign kind of breaks the content, since this para describes the second step of implementing a service. I will try to split it into bullet points such that it's clearer